Sources/Slider.swift

```diff
@@ -25,16 +25,22 @@ private func isAnimationAllowed() -> Bool {
     return !isSimulator && !ProcessInfo.processInfo.isLowPowerModeEnabled && !UIAccessibilityIsReduceMotionEnabled() && !isUnderHighload
 }
 
+public protocol SliderDelegate : class {
```

Comment:
Remove SliderDelegate and add closures
Slider: UIControl {
...
public var beginInteraction: (() -> (Void))?
public var endInteraction: (() -> (Void))?
...